bin/index.js

```diff
@@ -103,19 +113,26 @@ async function getFreePort(preferredPort = 5000) {
       }
     });
 }
-clear();
-const { open, port } = minimist(process.argv.slice(2), {
-  default: {
-    open: true,
-    port: 5000
-  }
-});
+
+/**
+ * Register Glicky commands
+ */
+glicky
+  .version(version, '-v, --version')
+  .usage('[options]')
+  .option('-o, --open <boolean>', 'Open Glicky in default browser', parseBoolean, true)
+  .option('-p, --port <number>', 'Open Glicky is specified port number', parseInteger, 5000)
```

Comment:
```suggestion
  .option('-p, --port <number>', 'Open Glicky on the specified port number', parseInteger, 5000)
```

Sorry for this annoying comment 😬

Reply:
My bad 🙈